spec/yaml/swiftnav/sbp/tracking.yaml

```diff
@@ -18,7 +18,46 @@ include:
 definitions:
 
  - TrackingChannelState:
-    public: True
+    short_desc: Satellite tracking channel state
```

Comment:
We're making everything public. We're making some packages stable, other not stable. We don't have a per-message stability, but we might need to.